lib/deqm_test_kit.rb

```diff
@@ -45,5 +46,6 @@ class Suite < Inferno::TestSuite
     group from: :measure_availability
     group from: :data_requirements
     group from: :submit_data
+    group from: :patient_everything
```

Comment:
Nit picking here, but I think it would be best to have patient_everything be the first test in the group (after Capability Statement). Since measure_availability -> data_req -> submit_data all need to be run in sequence and the url for patient_everything gets populated after running the capability statement suite anyway. This is just my opinion though, so your call. If you do choose to do this the suite number will need to be changed in all the rspec tests

Reply:
You know this better than I do so I'll trust your judgment